ember-handlebars-template.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'sprockets', '>= 3.3', '< 3.6'
+  spec.add_dependency 'sprockets', '>= 3.3', '< 4'
```

Comment:
I'll update dependent sprockets version to '< 4.1' after sprockets 4.0.0 released.